cli/command/defaultcontextstore.go

```diff
@@ -31,66 +29,32 @@ type ContextStoreWithDefault struct {
 	Resolver DefaultContextResolver
 }
 
-// EndpointDefaultResolver is implemented by any EndpointMeta object
-// which wants to be able to populate the store with whatever their default is.
-type EndpointDefaultResolver interface {
```

Comment:
Looks like this is used in buildx; let me check  how/why and if it's strictly needed 

```
+ xx-go build -ldflags '-X github.com/docker/buildx/version.Version=9e61f69 -X github.com/docker/buildx/version.Revision=9e61f69a5b83c3d6ee5545e488d5ad430c894276 -X github.com/docker/buildx/version.Package=github.com/docker/buildx -w -s' -o /usr/bin/buildx ./cmd/buildx
# github.com/docker/buildx/driver/kubernetes/context
driver/kubernetes/context/load.go:31:7: undefined: command.EndpointDefaultResolver
```